src/modules/game.js

```diff
@@ -1,33 +1,37 @@
+import renderScores from './render.js';
+
 class Game {
-  #scores;
+  #GAME_ID;
+
+  #URL_ENDPOINT;
 
   constructor() {
-    this.#scores = [
-      {
-        Name: 100,
-      },
-      {
-        Name: 20,
-      },
-      {
-        Name: 50,
-      },
-      {
-        Name: 78,
-      },
-      {
-        Name: 125,
-      },
-      {
-        Name: 77,
-      },
-      {
-        Name: 42,
-      },
-    ];
+    this.#GAME_ID = 'El0O6fUr7GdLZJ5Aoxwm';
+    this.#URL_ENDPOINT = `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${
+      this.#GAME_ID
+    }/scores`;
+  }
+
+  getScores() {
+    fetch(this.#URL_ENDPOINT)
+      .then((response) => response.json())
+      .then((data) => {
+        renderScores(data.result);
+      });
```

Comment:
- [ ] kindly consider using async and await with the fetch method to send and receive data. in order to fulfill the below requirement 👇 
- > Use async and await JavaScript features to consume the API.